src/ioctls/vcpu.rs

```diff
@@ -1049,6 +1049,47 @@ impl VcpuFd {
         Ok(reg_value)
     }
 
+    /// Notify the guest about the vCPU being paused.
+    ///
+    /// This signals to the host kernel that the specified guest is being paused by
+    /// userspace.  The host will set a flag in the pvclock structure that is checked
+    /// from the soft lockup watchdog.  The flag is part of the pvclock structure that
+    /// is shared between guest and host, specifically the second bit of the flags
+    /// field of the pvclock_vcpu_time_info structure.  It will be set exclusively by
+    /// the host and read/cleared exclusively by the guest.  The guest operation of
+    /// checking and clearing the flag must an atomic operation so
+    /// load-link/store-conditional, or equivalent must be used.  There are two cases
+    /// where the guest will clear the flag: when the soft lockup watchdog timer resets
+    /// itself or when a soft lockup is detected.  This ioctl can be called any time
+    /// after pausing the vcpu, but before it is resumed.
+    ///
+    /// See documentation for `KVM_KVMCLOCK_CTRL`.
```

Comment:
It looks like the description above contains the whole `KVM_KVMCLOCK_CTRL` documentation from https://www.kernel.org/doc/Documentation/virtual/kvm/api.txt. I would either try to summarize a bit the description and keep this line and maybe update it to:

`
See the documentation for KVM_KVMCLOCK_CTRL in the
[KVM API doc](https://www.kernel.org/doc/Documentation/virtual/kvm/api.txt).
`

or remove this line.

Reply:
Ok I'm gonna change that.